react/src/lib/components/DeckGLMap/layers/LassoLayer/lassoLayer.tsx

```diff
@@ -16,6 +16,8 @@ export interface LassoLayerProps<D> extends ExtendedLayerProps<D> {
     lineWidthScale: number;
     lineStyle: LineStyleAccessor;
     wellHeadStyle: WellHeadStyleAccessor;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    getSelectedWellsData: (pickingInfos: any[]) => void;
```

Comment:
Suggest having a more generic prop, eg. `handleSelection` to cover any selected data.